pkg/service/repair/service_repair_integration_test.go

```diff
@@ -432,6 +435,19 @@ func createKeyspace(t *testing.T, session gocqlx.Session, keyspace string, rf1,
 	ExecStmt(t, session, fmt.Sprintf(createKeyspaceStmt, keyspace, rf1, rf2))
 }
 
+func tryCreateTabletKeyspace(t *testing.T, session gocqlx.Session, keyspace string, rf1, rf2, tablets int) {
+	createKeyspaceStmt := "CREATE KEYSPACE %s WITH replication = {'class': 'NetworkTopologyStrategy', 'dc1': %d, 'dc2': %d}"
+	tabletStmt := " AND tablets = {'enabled': true, 'initial': %d}"
+	if tablets > 0 {
+		err := session.ExecStmt(fmt.Sprintf(createKeyspaceStmt+tabletStmt, keyspace, rf1, rf2, tablets))
+		if err == nil {
+			return
+		}
+		// Fallback as we don't know if tablets are enabled
```

Comment:
I'm not sure if I understand the fallback.
What happens with the executed statement above if tablets are not enabled ? Will the ExecStmt fail and return error ? 
If it's true, then you return from the method.

Or the next ExecStmt (without tablets part) will fail if the tablets are enabled ? But if the latter will fail, then the test will fail.

Reply:
> What happens with the executed statement above if tablets are not enabled ? Will the ExecStmt fail and return error ?

yes.

>  If it's true, then you return from the method.

yes, because method managed to create tablet keyspace.

> Or the next ExecStmt (without tablets part) will fail if the tablets are enabled ? But if the latter will fail, then the test will fail.

The second statement is correct for both vnode and tablets - it basically creates a keyspace with default type of replication. So if it fails, SM failed at creating regular keyspace and the test cannot proceed. 



In general it was annoying to check if tablets are enabled as it required to query experimental.tablets field from Scylla config. In Syclla 6.0 it was moved from experimental to regular fields. So it was annoying to check for that, and that's why I decided to use this simple approach for testing.

Reply:
fine